packages/runner-ct/src/app/Plugins.tsx

```diff
@@ -1,64 +1,57 @@
 import cs from 'classnames'
 import * as React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
 import State from '../lib/state'
 import { Hidden } from '../lib/Hidden'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import styles from './RunnerCt.module.scss'
-import { observer } from 'mobx-react'
+import { namedObserver } from '../lib/mobx'
 import { PLUGIN_BAR_HEIGHT } from './RunnerCt'
-import { UIPlugin } from '../plugins/UIPlugin'
+
+import styles from './RunnerCt.module.scss'
 
 interface PluginsProps {
   state: State
   pluginsHeight: number
-  pluginRootContainer: React.MutableRefObject<HTMLDivElement>
+  pluginRootContainerRef: React.MutableRefObject<HTMLDivElement>
 }
 
-export const Plugins = observer(
-  function Plugins (props: PluginsProps) {
-    function onClick (plugin: UIPlugin) {
-      return props.state.openDevtoolsPlugin(plugin)
-    }
-
-    return (
+export const Plugins = namedObserver('Plugins',
```

Comment:
Make sure that observer is not needed here. It was breaking the dev tools mounting – I removed it here #15612